packages/coordinator/src/tx-pool.ts

```diff
@@ -110,7 +110,7 @@ export class TxMemPool implements TxPoolInterface {
         memo: tx.memo
           ? {
               version: tx.memo.version,
-              data: Buffer.from(tx.memo, 'base64'),
+              data: Buffer.from(tx.memo.data, 'base64'),
```

Comment:
thx @sifnoc  I'm sorry that I missed this

Reply:
NVM!